src/styles/bar.css

```diff
@@ -5,7 +5,22 @@ body {
   height: 100vh;
 }
 
+
 [role=application] {
   height: initial;
   flex: 1 100vh;
 }
+
+
+[role=banner] {
+  --silver: #d6d8da;
```

Comment:
We normally enable a fallback using autoprefixer, but it seems to not work here, which is a little bit annoying. I'll investigate, thanks.